apiary.apib

```diff
@@ -3567,11 +3567,12 @@ Get a list of work orders.
 
     + Attributes (object)
         + filter (object, optional)
-            + state (enum[string], optional)
-                + Members
-                    + draft
-                    + finalized
-                    + sent
+            + state: `draft`, `sent` (array[string], optional) - The states you want to filter by
+                + (enum[string], required)
```

Comment:
Strong PP: you can use
```
            + state (array[enum[string]], optional)
                + Members
                    + draft
                    + finalized
                    + sent
```
The example enum values tend to mess up the preview in the live docs. Also the remark is not strictly needed (pretty obvious what it does).